src/test/java/org/apache/maven/plugins/gpg/GpgVersionTest.java

```diff
@@ -32,6 +34,8 @@ public void test()
         assertTrue( GpgVersion.parse( "gpg (GnuPG) 2.2.1" ).isAtLeast( GpgVersion.parse( "gpg (GnuPG) 2.2.1" ) ) );
         assertTrue( GpgVersion.parse( "gpg (GnuPG) 2.2.1" ).isAtLeast( GpgVersion.parse( "2.1" ) ) );
         assertTrue( GpgVersion.parse( "gpg (GnuPG/MacGPG2) 2.2.10" ).isAtLeast( GpgVersion.parse( "2.2.10" ) ) );
+        assertTrue( GpgVersion.parse( "gpg (GnuPG) 2.0.26 (Gpg4win 2.2.3)" )
```

Comment:
Why isAtLeast? I'm not sure I'm following this. It seems there should be an assertEquals here instead.

Reply:
i will change 3 because 3 should match equal version and one isAtLeast version.

Reply:
@elharo Done.

Reply:
Sometimes code helps more: look at https://github.com/apache/maven-gpg-plugin/blob/master/src/main/java/org/apache/maven/plugins/gpg/GpgSigner.java#L89-L104
You'll see there are features that were introduced in GPG 2.1. So you can do 2 things: use atLeast("2.1") or before("2.1"), but you will never be interested in equals("2.1"). 
The problem is: is `"gpg (GnuPG) 2.2.1"` equals to `"gpg (GnuPG/MacGPG2) 2.2.1"` ? There is no good answer, hence stick to atLeast/before or use compare